app_streaming.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import FastAPI
+from llama_cpp import Llama
+from transformers import pipeline
+from pydantic import BaseModel
+from fastapi.responses import StreamingResponse
+from typing import Optional
+
+# import uvicorn
+
+
+class Message(BaseModel):
+    text: str
+    temperature: Optional[float] = 0.3
```

Comment:
You can use the newer syntax:
```suggestion
    temperature: float | None = 0.3
```